src/main/resources/assets/galacticraft-rewoven/lang/en_us.json

```diff
@@ -414,10 +414,15 @@
   "category.rei.circuit_fabricator": "Circuit Fabricating",
   "category.rei.compressing": "Compressing",
   "galacticraft-rewoven:moon": "Moon",
-  "commands.galacticraft-rewoven.dimensiontp.success.multiple": "Successfully teleported %s entities to %s",
-  "commands.galacticraft-rewoven.dimensiontp.success.single": "Successfully teleported to %s",
-  "commands.galacticraft-rewoven.dimensiontp.failure.dimension": "Failed to locate dimension!",
-  "commands.galacticraft-rewoven.dimensiontp.failure.entity": "That is not a valid entity!",
+  "commands.galacticraft-rewoven.dimensiontp.success.multiple": "Teleported %s entities to %s",
+  "commands.galacticraft-rewoven.dimensiontp.success.single": "Teleported to %s",
+  "commands.galacticraft-rewoven.dimensiontp.success.pos": "Teleported to %s at (%d,%d,%d)",
+  "commands.galacticraft-rewoven.dimensiontp.failure.dimension": "Cannot locate dimension",
+  "commands.galacticraft-rewoven.dimensiontp.failure.already_in_dimension": "Already inside dimension %s",
+  "commands.galacticraft-rewoven.dimensiontp.failure.entity": "Invalid entity",
+  "commands.galacticraft-rewoven.gcrhouston.confirm": "Er, Houston, we have a problem... (Run this command again to confirm teleport)",
+  "commands.galacticraft-rewoven.gcrhouston.on_earth_already": "I don't need rescued!",
```

Comment:
```suggestion
  "commands.galacticraft-rewoven.gcrhouston.on_earth_already": "I don't need rescue!",
```

Reply:
I'll change it to "I don't need to be rescued!", it seems more natural, and more in line with what I was trying to go for initially.